packages/docs-app/src/common/films.tsx

```diff
@@ -136,17 +136,17 @@ export const renderFilm: ItemRenderer<IFilm> = (film, { handleClick, handleFocus
     if (!modifiers.matchesPredicate) {
         return null;
     }
-    const text = `${film.rank}. ${film.title}`;
     return (
         <MenuItem
+            text={highlightText(`${film.rank}. ${film.title}`, query)}
             active={modifiers.active}
+            // selected={modifiers.selected} // TODO: add selected prop to modifiers? or does this need to be handled outside of QueryList?
```

Comment:
Select2 has no controlled "selected item", so QueryList cannot set `modifiers.selected` for us.

The changes to `selected` semantics we're implementing in this PR make this standalone `renderFilm` function hard to re-use across the docs examples via `filmSelectProps`. We're going to have to remove it from `filmSelectProps` and copy it over to each of the example implementations in `filmSelect.tsx`, `omnibarExample.tsx`, and `suggestExample.tsx`.